core/runtime.rs

```diff
@@ -916,14 +1044,118 @@ impl JsRuntime {
             // Load is done.
             let module_id = load.root_module_id.unwrap();
             self.mod_instantiate(module_id)?;
-            match self.mod_evaluate(module_id) {
-              Ok(()) => self.dyn_import_done(dyn_import_id, module_id)?,
-              Err(err) => self.dyn_import_error(dyn_import_id, err)?,
-            };
+            self.dyn_mod_evaluate(dyn_import_id, module_id)?;
+          }
+        }
+      }
+    }
+  }
+
+  fn evaluate_pending_modules(&mut self) -> Result<(), AnyError> {
```

Comment:
Can you add a triple slash comment here describing what this does? What does "pending" mean in this case?

Reply:
Done, PTAL